### DISCUSSION TITLE
How to Transfer Non-Native Fungible Tokens (e.g., MOD) on Aptos?

### DISCUSSION THREAD

**Discussion by abdul-kabugu** [aptos-labs/aptos-developer-discussions#397] [Questions]
### Discord user ID

kabugu

### Describe your question in detail.

 I'm trying to transfer fungible tokens on Aptos from one address to another. I found an example in the docs for transferring native APT tokens, but I'm looking for an example or guidance on transferring other fungible tokens, such as MOD tokens. Does anyone have an example or can point me in the right direction? Thanks!

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by adityaaggarwal12** [comment]
to transfer legacy coins like mod you can use this - 
https://aptos.dev/en/build/smart-contracts/aptos-coin#transferring-coins:~:text=a%20DepositEvent.-,Transferring%20Coins,-A%20holder%20of

to transfer new fungiable asset standard coins - 
https://aptos.dev/en/build/smart-contracts/fungible-asset#transfer:~:text=a%20DepositEvent.-,Transfer,-An%20owner%20can
